runtime/service_integ_test.go

```diff
@@ -295,7 +295,7 @@ func TestMultipleVMs_Isolated(t *testing.T) {
 							fmt.Printf("stderr output from vm %d, container %d: %s", vmID, containerID, stderrOutput)
 						}
 
-						require.Equal(t, uint32(0), exitStatus.ExitCode())
+						require.Equal(t, uint32(0), exitStatus.ExitCode(), fmt.Sprintf("failed with %v", exitStatus.Error()))
```

Comment:
`require.Equal` should already be outputting both the expected and actual values; that's part of why we use it over `if exitStatus.ExitCode() != 0`.  Are you not seeing this output?